### ISSUE TITLE
Force linear/auto filter to certain textures + Hypno bow doesn't use the correct colors

### ISSUE THREAD

**Issue by AmyRoxwell** [Monkbreh/MHFU-Texture-Port#5]
There is a really big amount of textures that look super blurry or have issues when using auto or linear on the normal game, so most of the time you want to use nearest to avoid said issues, but certain textures depend of you using linear or auto to work properly.

Things like smoke, shadows and even certain skyboxes look super pixelated with nearest but look amazing with linear/auto (The same goes for a lot of textures that look great on nearest but no so much on linear),  specifying what filter the textures should use by using ppsspp replacement "force filter" feature on the texture .ini would allow us to have a better presentation for said textures 

Also, the hypno bow is using the wrong texture, it should be using instead the normal hypno texture instead of the breeding season hypno

![Screenshot_02242024_171910](https://github.com/Monkbreh/MHFU-Texture-Port/assets/89411895/d3c30920-1016-494a-9681-570d9b1e533d)
![Screenshot_02242024_171912](https://github.com/Monkbreh/MHFU-Texture-Port/assets/89411895/a9cec00a-ad02-4f61-acfb-8090b833779b)
![image](https://github.com/Monkbreh/MHFU-Texture-Port/assets/89411895/d797afb9-2dc6-4a68-a1f0-2ac75da16091)

Both of those things I'm planning to modify and put them as a PR.

**Comment by Monkbreh** [comment]
Interesting, i had no idea you can force filters on different textures. As for the hypno bow i already fixed that but i didnt update the github repo yet since am also working on adding the other equipment stuff.

**Comment by AmyRoxwell** [comment]
> Interesting, i had no idea you can force filters on different textures. 

I think it was a recent addition to it, you can see it on the .ini if you make a new one (sadly their website doesn't have anything about it for some reason), here is an example of it:
![image](https://github.com/Monkbreh/MHFU-Texture-Port/assets/89411895/2cf1dbec-98b2-483a-9113-8aad7f619f08)

> As for the hypno bow i already fixed that but i didnt update the github repo yet since am also working on adding the other equipment stuff.

Good to know, so just I don't clog the repo with issues, is the Queen's glass fixed? currently it's texture looks very dark compared of how it looks in vanilla or in any mh game tbh

Hash is 091e01c066932df35bde4f78 and it's a hunting horn

Vanilla:
![Screenshot_02222024_184438](https://github.com/Monkbreh/MHFU-Texture-Port/assets/89411895/e32a26b9-f877-4c08-8b34-ec67cf5f3702)
4U and GEN:
![image](https://github.com/Monkbreh/MHFU-Texture-Port/assets/89411895/832e22a1-ac9c-49b8-80b4-b66a34716111)

Texture Pack:
![Screenshot_02222024_184445](https://github.com/Monkbreh/MHFU-Texture-Port/assets/89411895/b54e7992-c999-47b6-8800-b2e728b0889a)

**Comment by Monkbreh** [comment]
Yeah am revising alot of these textures, 2nd gen textures are less saturated and dark compared to other games. Am also gonna mess around with the texture filter thing, thx for informing me on that. Its really nice that you can have different filters for different  textures cause aside from UI and effects alot of stuff tend to look better with nearest.

**Comment by AmyRoxwell** [comment]
> Yeah am revising alot of these textures, 2nd gen textures are less saturated and dark compared to other games.

I tried to search for that texture on frontier files but I couldn't find it, we could probably get away with changing the colors with the ones on 4U, so it has it's color back :thinking: 

> thx for informing me on that. Its really nice that you can have different filters for different textures cause aside from UI and effects alot of stuff tend to look better with nearest.

Yep! Specially I was thinking on the UI with the texture pack, it looks pretty nasty with nearest but looks perfectly fine on linear, we could probably force a lot of stuff to just use nearest or tell the user that this pack is intended to use with nearest as the texture filter (due to linear being apply way less).

I will began rn for all the textures I can see in a first glance that require linear to look correctly

**Comment by AmyRoxwell** [comment]
So, I just discover you can just combine the replacement texture argument + the filter one, like this 
![image](https://github.com/Monkbreh/MHFU-Texture-Port/assets/89411895/07ca3361-8fc6-461d-8f25-70796ea2e50f)

and then you can just have it apply both at the same time, soft lighting >:3
![Screenshot_02252024_015832](https://github.com/Monkbreh/MHFU-Texture-Port/assets/89411895/de52d402-5ac3-41f5-9eb1-601d3fdd245a)

**Comment by AmyRoxwell** [comment]
Alright, good news and bad news:
the good is that I was able to figure out how the wyvern flame trail texture works and now it displays correctly (the smoke trail they use on frontier doesn't play well, making an ugly trail in a square shape)
![Screenshot_03022024_183120](https://github.com/Monkbreh/MHFU-Texture-Port/assets/89411895/4c917e88-8cd1-4c9d-a0a2-cd36d81a9c5f)

The culprit was this smoke texture, it doesn't seem to be use on this game and only on frontier (need to triple check this, also red background so you can actually see it)
![image](https://github.com/Monkbreh/MHFU-Texture-Port/assets/89411895/4beb0f93-3431-4a2a-9c6b-0834e342cc5d)

the texture on FU and dos looks like this instead.
FU: 
![095ca7a0db88650debe33f60](https://github.com/Monkbreh/MHFU-Texture-Port/assets/89411895/b92bb732-3c0f-4c2d-8e77-e294a52de0d8)
Dos: (ignore the weird alpha)
![THEONE](https://github.com/Monkbreh/MHFU-Texture-Port/assets/89411895/5f8cd5d7-8ae6-4160-a8ef-7215feb8d571)

Notice how both dont includes that smoke trail and the fire balls are instead on that place, so I just did a quick replacement and  this was the result
![Fireball-Trail](https://github.com/Monkbreh/MHFU-Texture-Port/assets/89411895/2f373f40-b76f-4921-8463-5ffb46392fa5)


Before:
![Screenshot_03032024_115813-1](https://github.com/Monkbreh/MHFU-Texture-Port/assets/89411895/c3b047ef-17f9-41bf-ab99-8c49109c380f)
After: 
![image](https://github.com/Monkbreh/MHFU-Texture-Port/assets/89411895/d90735b8-9483-41e9-a17c-f4065660e3d4)

Now, I could submit this as a pr...But here are the bad news, even if a lot of monsters reuse this texture, very monster has its own fireball texture hash, so It is going to take me sometime to get all the hashes and try to just redirect them to the modified texture (I could move it to EFX and that way have it easy access), hopefully it doesn't take that long!

**Comment by Monkbreh** [comment]
Huh good eye, i would fix this but i see your already on it. Anyways my lazy arse should've sorted the monsters into different folders instead of dropping everything in one place. (Something i would address in the next update). You can create an new EFX folder in the monsters folder or use the preexisting one.

**Comment by AmyRoxwell** [comment]
> Huh good eye, i would fix this but i see your already on it. Anyways my lazy arse should've sorted the monsters into different folders instead of dropping everything in one place. (Something i would address in the next update). You can create an new EFX folder in the monsters folder or use the preexisting one.

Yeag, I want to first test if there is an edge case with one or two monsters with its fireball attacks, so I would hold before making a pr!!
As for the EFX, I would just put it on the one already existing also, good to know you will be putting monster textures on folders, I was going to do it but I prefer waiting for what you do and just focus on stuff that already is organized haha!

**Comment by AmyRoxwell** [comment]
Hello, It's me again. I was wondering if the new update for this mod is still a planned thing so I can continue testing. Thanks!

**Comment by Monkbreh** [comment]
Yeah, am going to push an update for the mod with labelled file structures so its more coherent and easier to identify textures. I've been busy so hopefully you can wait a little bit more :)

**Comment by AmyRoxwell** [comment]
> Yeah, am going to push an update for the mod with labelled file structures so its more coherent and easier to identify textures. I've been busy so hopefully you can wait a little bit more :)

Yeag, No problem. I was just worried something happened to you after this long time, glad to know you are still here!

**Comment by AmyRoxwell** [comment]
> Yeah, am going to push an update for the mod with labelled file structures so its more coherent and easier to identify textures. I've been busy so hopefully you can wait a little bit more :)

Hello, Wanted to check out on the progress of said update, It's been a while since any news.
